manifests/cf-manifest/deployments/030-cf-jobs.yml

```diff
@@ -763,7 +763,7 @@ properties:
   support_address: "http://support.cloudfoundry.com"
   description: null
   ssl:
-    skip_cert_verify: true
+    skip_cert_verify: false
```

Comment:
I've created https://github.com/alphagov/paas-cf/pull/282. There shouldn't be any merge conflicts.